test/emitter.test.ts

```diff
@@ -59,6 +60,25 @@ describe("Emitter", () => {
     emitter.dispose()
   })
 
+  it("should log an error if something goes wrong", async () => {
```

Comment:
Should this test check that the logger was called? It doesn't seem to make any assertions right now.

Reply:
woah weird, maybe it got sucked away when i rebased. good catch!

Reply:
🤦‍♂️ rebase gone wrong. this is a duplicate test lol